src/main/webapp/jsp/instructorFeedbackResultsByRecipientGiverQuestion.jsp

```diff
@@ -4,7 +4,7 @@
 <%@ taglib tagdir="/WEB-INF/tags/instructor/results" prefix="results" %>
 <c:set var="jsIncludes">
     <script type="text/javascript" src="/js/feedbackResponseComments.js"></script>
-    <script type="text/javascript" src="/js/instructorFeedbackResultsAjaxByRQG.js"></script>
+    <script type="text/javascript" src="/js/instructorFeedbackResultsAjaxByRGQ.js"></script>
```

Comment:
Hopefully now that we are minimising the number of different ways the page can load, this sort of bugs can be detected more quickly.